_specification-v1/sass.md

```diff
@@ -121,7 +121,7 @@ Silent mode means a component's Sass will compile to an empty string, but provid
 
 Components that make use of styles defined in other components which support silent mode **must** use those styles silently, e.g. for a component `o-foo` which depends on `o-bar`:
```

Comment:
Note: This section of the specification needs updating to [remove the silent mode variable in favour of the primary mixin](https://github.com/Financial-Times/origami/pull/121). V2 component **must not** output styles by default on `@import`.